src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/helper/EnvelopeCreator.java

```diff
@@ -50,6 +51,35 @@ public static Envelope envelope(String jurisdiction, Status status) throws Excep
             timestamp,
             timestamp,
             UUID.randomUUID() + ".zip",
+            Classification.EXCEPTION,
+            false,
+            scannableItems(),
+            payments(),
+            nonScannableItems()
+        );
+
+        envelope.setStatus(status);
+        envelope.setContainer("SSCS");
+
+        return envelope;
+    }
+
+    public static Envelope envelopeUrgent() throws Exception {
+        return envelope("SSCS", Status.CREATED);
```

Comment:
I think you should call `envelopeUrgent` here

Reply:
Well spotted. Thanks